AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestExecutionStore.kt

```diff
@@ -0,0 +1,30 @@
+package dev.androidx.ci.testRunner
```

Comment:
File is missing a copyright header

Reply:
@liutikas Is this generated from any tool, or can I just copy it from a different file?

Reply:
for the future, you should be able to run ktFormat to fix them (and other files) automatically.